tests/dummy/app/controllers/demo/select.js

```diff
@@ -66,6 +82,11 @@ export default Ember.Controller.extend({
 
   vegToppings: Ember.computed('toppings.[]', function() {
     return this.get('toppings').filter((item) => item.category === 'veg');
-  })
+  }),
+  actions: {
+    searchKeyPressed(e) {
+      e.stopPropagation();
```

Comment:
Why are you stopping propagation?
Maybe if we use `<input` instead of `{{input` this action may do something meaningful like `this.set('searchTerm', value);`?